src/Education.UnitTests/SimpleRuleHelpXamlBuilderTests.cs

```diff
@@ -133,15 +131,17 @@ private static bool ProcessResource(string fullResourceName)
                 var data = ReadResource(fullResourceName);
                 var jsonRuleInfo = LocalRuleMetadataProvider.RuleInfoJsonDeserializer.Deserialize(data);
 
-                var doc = testSubject.Create(jsonRuleInfo);
-
-                // Quick sanity check that something was produced
-                // Note: this is a quick way of getting the size of the document. Serializing the doc to a string
-                // and checking the length takes much longer (around 25 seconds)
-                var docLength = doc.ContentStart.DocumentStart.GetOffsetToPosition(doc.ContentEnd.DocumentEnd);
-                Console.WriteLine($"{jsonRuleInfo.FullRuleKey}: size = {docLength}");
-                docLength.Should().BeGreaterThan(30);
-
+                if (!string.IsNullOrWhiteSpace(jsonRuleInfo.Description))
```

Comment:
some hotspot rules do not have the description field. (they have descriptionsections filled)